MoviesExample/app/src/main/java/com/novoda/demo/movies/ImageURLBinding.java

```diff
@@ -0,0 +1,15 @@
+package com.novoda.demo.movies;
+
+import android.databinding.BindingAdapter;
+import android.widget.ImageView;
+
+import com.bumptech.glide.Glide;
+
+public class ImageURLBinding {
```

Comment:
Class containing a custom binding. The custom binding method is annotated with `@BindingAdapter` and can be in any class of your code. It's convenient to have one class for each custom binding or one single class for all of them